src/main/resources/resources/logisim/strings/std/std_ru.properties

```diff
@@ -774,6 +774,7 @@ TTL7414 = 7414: шестнадцатеричный инвертор (шмитт-
 TTL74161 = 74161: 4-bit sync counter with clear
 # ==> TTL74163 =
 TTL74165 = 74165: 8-битный регистр параллельного последовательного сдвига
+TTL74166 = 74166: 8-битный регистр параллельного последовательного сдвига с очистить
```

Comment:
If possible, please be more precise to state that the clear is asynchronous.

Reply:
done